icu4c/source/i18n/unicode/unumberrangeformatter.h

```diff
@@ -0,0 +1,431 @@
+// © 2020 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+
+#include "unicode/utypes.h"
+
+#if !UCONFIG_NO_FORMATTING
+#ifndef __UNUMBERRANGEFORMATTER_H__
```

Comment:
The guard should be between the copyright and the first include, like in other files. Remember to move the corresponding endif+comment to the last line, too.

Reply:
Seems that we're not consistent here (I spot several files with my order), but fixed via commit amendment.